src/main/java/com/github/hornta/race/Racing.java

```diff
@@ -267,6 +267,18 @@ private void setupCommands() {
       )
       .requiresPermission(Permission.COMMAND_SET_TYPE.toString())
       .requiresPermission(Permission.RACING_MODIFY.toString());
+      
+    carbon
+      .addCommand("racing setStartOrder")
+      .withHandler(new CommandSetStartOrder(racingManager))
+      .withArgument(raceArgument)
+      .withArgument(
+        new CarbonArgument.Builder("stat")
```

Comment:
I think renaming stat to order would be more meaningful.
```suggestion
        new CarbonArgument.Builder("order")
```